packages/dcos-diagnostics/buildinfo.json

```diff
@@ -2,11 +2,11 @@
   "requires": ["exhibitor"],
   "single_source" : {
     "kind": "git",
-    "git": "https://github.com/dcos/3dt.git",
-    "ref": "6b560880d5a2419b8bad387d06d9355d3ad207ab",
+    "git": "https://github.com/dcos/dcos-diagnostics.git",
+    "ref": "ec7fb9d99fcb77120978f7dfbe382aca7702102d",
     "ref_origin": "master"
   },
-  "username": "dcos_3dt",
+  "username": "dcos_diagnostics",
```

Comment:
Is there a risk that files previously owned by the `dcos_3dt` user will no longer be accessible?